.github/workflows/ci-nym-vpn-app-rust.yml

```diff
@@ -55,9 +55,9 @@ jobs:
 
       - name: Install rust toolchain (Windows)
         if: contains(matrix.os, 'windows')
-        uses: brndnmtthws/rust-action-rustup@v1
+        uses: dtolnay/rust-toolchain@master
         with:
-          toolchain: stable
+          toolchain: 1.82.0
```

Comment:
I assume this is a temporary change just to unblock the release builds?

Reply:
yes!